### PR TITLE
Stage0-Stage1 - update and fix Git & Markdown tasks description

### PR DESCRIPTION
**Pull request by sergiozeppo**
#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [x] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [x] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [x] ✏️ Fixed a typo or grammatical error
- [x] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

- **Brief Overview:**
  - Update the description of the step-by-step task deployment process in both languages Stage0-Stage1.
  - Fix the link to the school logo
  - Fix links to courses and sources
  - Fix several typos
- **Implementation Approach:**

#### Additional Information

- **Screenshots/Links:**
- [ ] Related Issues:

#### Checklist

- [ ] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [ ] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: tasks/cv/cv-stage0.md [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 **Максимальный балл за задание +120**
 
-- студент засабмитил в [rs app](https://app.rs.school/course/student/cross-check-submit?course=js-fe-preschool-2022q2) ссылку на своё CV, задеплоенное на `gh-pages` +10
+- студент засабмитил в [rs app](https://app.rs.school/course/student/cross-check-submit?course=js-fe-preschool-2024q2) ссылку на своё CV, задеплоенное на `gh-pages` +10
```
Comment (WiiJoy) [inline tasks/cv/cv-stage0.md:40]: Мб убрать линки на сабмит в конкретных курсах, дабы для каждого нового потока не вносить правку в этот линк?)

File: tasks/cv/en/cv-stage0.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ In this task, you are required to create a CV webpage. The content of this webpa
 
 **Maximum score for the task: +120**
 
-- The student submits a link to their CV deployed on `gh-pages` in the [rs app](https://app.rs.school/course/student/cross-check-submit?course=js-fe-preschool-2022q2) (+10 points).
+- The student submits a link to their CV deployed on `gh-pages` in the [rs app](https://app.rs.school/course/student/cross-check-submit?course=js-fe-preschool-2024q2) (+10 points).
```
Comment (WiiJoy) [inline tasks/cv/en/cv-stage0.md:35]: Аналогично

File: tasks/cv/cv-stage0.md (hunk 1 above)
Comment (sergiozeppo) [reply tasks/cv/cv-stage0.md]: > Мб убрать линки на сабмит в конкретных курсах, дабы для каждого нового потока не вносить правку в этот линк?)

Привет. Согласен, поправил в обоих случаях.

**Review by stardustmeg — APPROVED** [review]

**Comment by sergiozeppo** [comment]
@Luffi2539 @natein @muannna @wiijoy
Hi everyone! 
The FE2024 course starts in a few days and the first assignment is - this. 
It's been on a review for a long time, and seems to be there are no remarks on it.  
Review it, please.

**Review by WiiJoy — APPROVED** [review]

**Review by muannna — APPROVED** [review]

**Pull request merged**